example_config

```diff
@@ -30,6 +30,9 @@
     "comment": "Fetch GCS Buckets",
     "fetch_file_names": false
```

Comment:
Let's just have it as a sub flag under storage_buckets similar to fetch_file_names.

Reply:
Done